site_prism.gemspec

```diff
@@ -5,7 +5,7 @@ require './lib/site_prism/version'
 Gem::Specification.new do |s|
   s.name        = 'site_prism'
   s.version     = SitePrism::VERSION
-  s.required_ruby_version = '>= 2.2'
+  s.required_ruby_version = '>= 2.3'
```

Comment:
align

Reply:
To get this file all aligned correctly is one that i'll let someone else tackle (If you wanna tackle it go for it), as there's a few different ways you can do it.